### PR TITLE
Milestone11: Contact validation with js

### PR DESCRIPTION
**Pull request by RuriEnciso**
### **Milestone11: Contact validation with js**
In this milestone we: 

- Added contact validation with js.
- Customized contact form.
- Fixed linter errors.

### CODE REVIEW

**Review by taldr27 — COMMENTED** [review]
# Status: Invalidated 🔴
Hi @RuriEnciso, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the proper linters set up. This milestone requires you to use the `html-css-js` linter which is a new setup than the one you've been using so please visit the [linters config here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) and update your repo accordingly.

After you've set up the linters you should see 5 checks (like the one below) instead of 4 checks.
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready. 🙏

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit. gif_

File: validation.js [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+    // If the field is empty,
+    // display the following error message.
+    emailError.textContent = 'You need to enter an email address.';
+  } else if (email.validity.typeMismatch) {
+    // If the field doesn't contain an email address,
+    // display the following error message.
+    emailError.textContent = 'Entered value needs to be an email address.';
+  } else if (/[A-Z]/.test(email.value)) {
+    emailError.textContent = 'Entered value has to be in lower case';
+  }
```
Comment (AnselemOdims) [inline validation.js:22]: - [ ] Good job working on this, but as per the requirement, the error is supposed to appear when a user tries to submit the form. Please kindly run the validation check when a user tries to submit the form.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @RuriEnciso , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @RuriEnciso , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi team @RuriEnciso, @danielf-canon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README. 🤓
✅ Linters tests passing. 👍
✅ Correct use of GitHub Flow. 🙌
✅ Form validations working correctly.🎯

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**